modules/graphql_1/main.tf

```diff
@@ -31,7 +37,10 @@ data "aws_iam_policy_document" "gql_task_execution_policy_document" {
       "logs:DescribeLogStreams",
       "logs:PutLogEvents"
     ]
-    resources = ["*"]
+    resources = [
+      "arn:aws:logs:${local.region}:${local.account_id}:log-group:/aws/lambda/${var.prefix}-graphql-invoke:*",
```

Comment:
The graphql-invoke lambda is just something I wrote for testing, and is not the real frontend. Does it raise errors if it doesn't have log permissions?

Reply:
updated to use the correct resource.